tony-core/src/main/java/com/linkedin/tony/Utils.java

```diff
@@ -416,5 +421,24 @@ public static void printWorkerTasksCompleted(AtomicInteger completedWTasks, long
     LOG.info("Completed worker tasks: " + completedWTasks.get() + " out of " + totalWTasks + " worker tasks." );
   }
 
+  public static Configuration addResourceToConfig(Configuration tonyConf, CommandLine cliParser) {
```

Comment:
this refactoring looks weird. it is fine to keep this portion of code inside `init()` function.

Reply:
Agreed. I think this could be a private method in `TonyClient` called `initTonyConf()`. What the method does seems specific to the TonyClient (preparing its Configuration) rather than a general utility method.

Reply:
👍